content/rc/rc-quickstart.md

```diff
@@ -26,20 +26,31 @@ If you already have a subscription, see [Manage subscriptions]({{< relref "/rc/s
 
 To create a new account with a free subscription and database:
 
-1. On the Redis Cloud [admin console](https://app.redislabs.com/#/login), select [Sign up](https://redis.com/try-free/).
+1. Go to the [Sign up](https://redis.com/try-free/) page.
 
-1. Enter your information in the form and select **Get Started**, or sign up with Google or Github.
+1. There are two options available to get started with Redis Cloud:
+   * Enter your information in the form and select **Get Started**.
+   * Sign up with **Google** or **Github**.
```

Comment:
```suggestion
   * Sign up with **Google** or **Github**.
```
If at all possible, we should make these links to the relevant google and github pages.

Reply:
I'm not sure this is possible. If I click the social login buttons, it just brings me to a social login screen that's specific to my accounts.